draft-jhoyla-tls-key-injection.md

```diff
@@ -53,27 +53,25 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
-# PSK Injection
+# Key Schedule Injection
+
+This section describes ways in which additional secrets can be injected into
+the TLS 1.3 key schedule.
+
+## Early Secret Injection
 
 TLS provides exporter keys that allow for other protocols to provide
 data authenticated by the TLS channel. This can be used to bind a protocol to a
 specific TLS handshake, giving joint authentication guarantees.
+In a similar way, one may wish to introduce externally authenticated and 
+pre-shared data to the early secret derivation. This can be used to bind external 
+protocols to the TLS protocol. 
```

Comment:
I'm wondering here if we should say "bind TLS to the external protocol". Bindings can have a direction (inward, outward, or both), and matching the order used in the previous sentence might stop people assuming that all bindings are from the external protocol to the TLS channel. But maybe I'm overthinking this.